tests/e2e/e2e_suite_test.go

```diff
@@ -380,6 +380,7 @@ func getElementalOperatorLogs() {
 	)).To(Succeed())
 
 	for _, pod := range podList.Items {
+		kubectl.Run("wait", "--for=condition=Ready", "pod", pod.Name, "-n", pod.Namespace, "--timeout=30s")
```

Comment:
Test are executed in a random order and I noticed that if the downgrade/upgrade tests is the last one this step here is likely to fail because last thing the downgrade/upgrade test does is upgrading again to the testing version and when this code was executed the elemental-operator pod was not yet running (was in `ContainerCreating` phase).